cypress.config.ts

```diff
@@ -6,9 +6,11 @@ module.exports = defineConfig({
   experimentalWebKitSupport: true,
   e2e: {
     setupNodeEvents(on, config) {
-      allureWriter(on, config);
-      return config;
+      // allureWriter(on, config);
```

Comment:
why do we need to comment this out? 
I think we need these two lines for the report

Reply:
updated!